accessibility-oscilloscope/overlay/usr/bin/wacom-btn

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
-DEVICE=$(libinput list-devices | grep "Wacom Intuos BT M Pad" -A1 | awk '/Kernel:/ {print $2}')
+DEVICE=/dev/input/event1
```

Comment:
if this has worked every time so far i'm ok with hard coding this, mostly just curious why libinput parses one event correctly but not the other.

most of the concern is if the event ordering is nondeterministic and things just hang parsing totally wrong output.